guide/sequelize/currency.md

```diff
@@ -320,9 +320,9 @@ return message.channel.send(`Successfully transferred ${transferAmount}💰 to $
 ```
 As a bot creator, you should always be thinking about how to make the user experience better. Good UX makes users less frustrated with your commands. If your inputs are different types, don't make them memorize which parameters come before the other.
 
-We want to allow users to do both `!transfer 5 @user` and `!transfer @user 5`. So what we're going to do is grab the first non-mention text in the command. In the second line of the above code, we split the command by spaces, and then we look for anything that doesn't match a mention, and we'll assume that's the transfer amount. Then we do some checking to make sure it's a valid input. You can also do error checking on the transfer target, but we won't include that here because of its triviality.
+You'd ideally want to allow users to do both `!transfer 5 @user` and `!transfer @user 5`. So you can grab the first non-mention text in the command. In the second line of the above code: split the command by spaces and look for anything that doesn't match a mention; you can assume that's the transfer amount. Then do some checking to make sure it's a valid input. You can also do error checking on the transfer target, but we won't include that here because of its triviality.
```

Comment:
This part sounds off, but it might be because it was also butchered earlier in the grammarly PR.
```suggestion
You'd ideally want to allow users to do both `!transfer 5 @user` and `!transfer @user 5`. To get the amount, you can grab the first non-mention text in the command. In the second line of the above code: split the command by spaces and look for anything that doesn't match a mention; you can assume that's the transfer amount. Then do some checking to make sure it's a valid input. You can also do error checking on the transfer target, but we won't include that here because of its triviality.
```